include/MomentumAdvDiffElemKernel.h

```diff
@@ -1,5 +1,5 @@
 /*------------------------------------------------------------------------*/
-/*  Copyright 2014 National Renewable Energy Laboratory.                  */
+/*  Copyright 2014 Sandia Corporation.                                    */
```

Comment:
Can we split off the pull request such that the copyright commits are added with the other one you have open? It seems cleaner that way.

Reply:
I have squashed and would like to use this commit to drive the discussion. In reality, there are many issues with copyright. We seem to be copying them around by mistake. Moreover, despite your desire to keep the development fork clean, ownership of many feels was completely removed. I think we knew that a while ago, however, it seems more prevalent now that I am back tracking to look at history.